File: C/6-2/4.c
#include <stdio.h>

int main(void)
{
	int n1, n2, n3;
	int min, max;
	scanf("%d %d %d", &n1, &n2, &n3);

	max = n1 < n2 ? (n2 < n3 ? n3 : n2) : (n1 < n3 ? n3 : n1);
	min = n1 > n2 ? (n2 > n3 ? n3 : n2) : (n1 > n3 ? n3 : n1);

	printf("min: %d \nmax: %d\n",min,max);

	return 0;
}

File: C/6-2/4.py
num = input().split()
a=num[0]
b=num[1]
c=num[2]

if int(a)>int(b) and int(a)>int(c):
    max=a
elif int(b)>int(a) and int(b)>int(c):
    max=b
else:
    max=c
if int(a)<int(b) and int(a)<int(c):
    min=a
elif int(b)<int(a) and int(b)<int(c):
    min=b
else:
    min=c


print("min: ",end='')
print(min)
print("max: ",end='')
print(max)
